src/Microsoft.Framework.PackageManager/Building/BuildManager.cs

```diff
@@ -208,7 +208,11 @@ public bool Build()
 
             sw.Stop();
 
-            projectDiagnostics.ForEach(d => LogWarning(d.FormattedMessage));
+            if (projectDiagnostics.Any())
+            {
+                LogWarning("");
```

Comment:
We should never log an empty line - that assumes a text-based logger. Instead you could consider logging some sort of "header" such as "found X diagnostic errors" or whatever.